README.md

```diff
@@ -2208,7 +2208,8 @@ Configuration of on-screen keyboard (if available).
 |`keyboardLayout`|KeyboardLayout|False|Desired keyboard layout.|
 |`keypressMode`|KeypressMode|False|Desired keypress mode.                If omitted, this value will be set to RESEND_CURRENT_ENTRY.            |
 |`limitedCharacterList`|String[]|False|Array of keyboard characters to enable.|
-|`autoCompleteText`|String|False|Allows an app to prepopulate the text field with a suggested or completed entry as the user types|
+|`autoCompleteText`|String|False|Deprecated, use autoCompleteList instead.|
+|`autoCompleteList`|String[]|False|Allows an app to prepopulate the text field with a list of suggested or completed entries as the user types.                 If empty, the auto-complete list will be removed from the screen.            |
```

Comment:
The [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0108-AutoCompleteList.md) includes the following in the param description:
`Set to 0 to remove the auto-complete list from the screen`